book/modules/appendix1-exercises.tex

```diff
@@ -17,18 +17,50 @@
 
 			\item $\frac{x}{y}=1$
 		\end{enumerate}
+		\begin{solution}
+			\begin{enumerate}
+				\item It is a linear equation.
+
+				\item It is a linear equation.
+
+				\item It is not a linear equation, because it involves $\cos(y)$,
+					the cosine of a variable.
+
+				\item It is not a linear equation, because it involves $xy$, a product
+					of two variables.
+
+				\item It is a linear equation.
+
+				\item It is not a linear equation, because it involves
```

Comment:
This equation is actually equivalent to `(log 2-1)x+(log 2)y=0`. Do we consider this as a linear equation?